reference/errors/error_codes.md

```diff
@@ -54,6 +54,10 @@ An index with this UID was not found, check out our guide on [index creation](/l
 
 The requested index already has a primary key that cannot be changed.
 
+## `index_primary_key_multiple_candidates_found`
+
+[Primary key inference](/learn/core_concepts/primary_key.md#meilisearch-guesses-your-primary-key) failed as the index contains multiple fields ending with `id`. Use the [update index endpoint](/reference/api/indexes.md#update-an-index) to manually set a primary key or add `id` to one of your fields so it can be used as the primary key during inference. We recommend manually setting the primary key.
```

Comment:
AFAIK adding `id` won't work in this case because we already have multiple candidates, and `id` is not given any priority.

As it probably doesn't make sense for users to remove fields from their documents, I'd say that manually specifying the field representing the primary key is their only recourse when there are multiple candidates.

Adding `id` to one of the fields **does** work in the `index_primary_key_no_candidate_found` case, though.